### ISSUE TITLE
Staticfile is not documented?

### ISSUE THREAD

**Issue by spinus** [flynn/flynn#2405]
Going through the documentation I could easily find how to deploy python, php or node but there was nothing about static files even if this is supported.

Is there anything about that in the docs? Maybe I didn't look in correct place.

**Comment by philiplb** [comment]
You might just deploy it as PHP application. Then it's served via an Apache or ngix with PHP support which you don't have to use.

**Comment by kwando** [comment]
I'm not sure where the documentation on this is. But to make it work just have to have a empty file named ```Staticfile``` in your project root and the correct and the static file build pack while pick it up.

**Comment by spinus** [comment]
sorry, maybe I was not precise. I managed it to work, I found staticfile example in one of flynn repositories.

If you look on https://flynn.io/docs/how-to-deploy-go page, on the left you have instructions how to deploy some stuff, but there is no manual for staticfiles.

I actually found that flynn can handle more "buildpacks" than documentation says, I found https://github.com/flynn/flynn/blob/master/slugbuilder/builder/buildpacks.txt which list probably all of them (and some are not documented as well)
